### PR TITLE
Feature/jira 100-PR for Feature1&2

### PR DESCRIPTION
**Pull request by saxenasandeep**
Please publish the login PR

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,12 @@
...
+
+Gradle build demo sample project
+
+This is Sample Demo app. Development in Progress
+
+This is additional line
+
+This is second session for Git
+
+This is third feature
```
Comment (sanrocks123) [inline README.md:12]: Please remove this line from code

**Review by sanrocks123 — CHANGES_REQUESTED** [review]
Please addressed this commments

File: README.md (hunk 1 above)
Comment (saxenasandeep) [reply README.md:12]: ok. I will do it

**Review by saxenasandeep — COMMENTED** [review]
ok

**Review by sanrocks123 — APPROVED** [review]

**Comment by sanrocks123** [comment]
ready to prod release

**Comment by sanrocks123** [comment]
add

**Pull request merged**